testing/dart/gesture_settings_test.dart

```diff
@@ -0,0 +1,54 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui';
+
+import 'package:litetest/litetest.dart';
+
+void main() {
+  test('GestureSettings has a reasonable toString', () {
+    const GestureSettings gestureSettings = GestureSettings(physicalDoubleTapSlop: 2.0, physicalTouchSlop: 1.0);
+
+    expect(gestureSettings.toString(), 'GestureSettings{physicalTouchSlop: 1.0, physicalDoubleTapSlop: 2.0}');
+  });
+
+  test('GestureSettings has a correct equality', () {
+    // don't refactor these to be const, that defeats the point!
+    double value = 2.0;
+    final GestureSettings settingsA = GestureSettings(physicalDoubleTapSlop: value, physicalTouchSlop: 1.0);
+    final GestureSettings settingsB = GestureSettings(physicalDoubleTapSlop: value, physicalTouchSlop: 3.0);
+    final GestureSettings settingsC = GestureSettings(physicalDoubleTapSlop: value, physicalTouchSlop: 1.0);
+    value++;
```

Comment:
Nit: Is the intention to use the value after?

Reply:
just trying to defeat the `const` checks. If I don't modify value then it says make it const, but then I have to make GestureSettings const, and the whole thing turns into a test of Dart's object canonicalization.

Reply:
Maybe leave a brief comment on this line for the future reader?

Reply:
Done